### DISCUSSION TITLE
Introductions! 👋

### DISCUSSION THREAD

**Discussion by edakturk14** [BuidlGuidl/batch3.buidlguidl.com#1] [Announcements]
Hey everyone - It's Eda here, I've been part of the BuidlGuidl for almost 1.5 years! I'm a developer and I spend a lot of time [writing](https://mirror.xyz/edatweets.eth) about different topics and concepts in crypto. I enjoy working on developer education and creating content/documentation.

Can't beleive that its already Batch3! See you on Telegram!

**Comment by Ahmedborwin** [comment]
Hi - Eda! My name is Ahmed. I come from a non-programming background. Have been learning to code over the last 18 months. Taken part in two hackathons. Looking forward to taking part in this batch and getting to know other people in the space!

**Comment by vitorpy** [comment]
Hey folks! I have been a coder for a long time and I worked on a Ethereum wallet a couple years ago. I'm coming back to Ethereum after a stint in big tech and going through BuidlGuidl to exercise my skills.

**Comment by Pabl0cks** [comment]
Hi everyone!

Pablo here. I've been part of the BuidlGuidl for almost a year! I've been a product owner/project manager in tech for almost 15 years, but I didn't reconnect with programming until a couple of years ago. 

I've been in crypto since 2017, but it was last year when I started learning blockchain development thanks to SpeedRunEthereum and BG. 
Love building things, testing, documenting, thinking about new features, and being surrounded by such brilliant minds 👾♥

Very excited for this batch, as it will be my first one actively helping. If you have any questions, ask anytime here or in TG! Hope we learn a lot and have fun.

**Comment by MatheusDaros** [comment]
Hello Guild

I'm Matheus Pagani, Developer, Teacher and Manager in the Web3 space since 2016.
I've done a lot of dev education with bootcamps and workshops, and I'm super glad to be part of BuidlGuidl Batch 3 🙏🏼 

Let's do some code 🚀
